peers/peerpool_test.go

```diff
@@ -336,20 +345,23 @@ func TestPeerPoolNotAllowedStopping(t *testing.T) {
 	server := &p2p.Server{
 		Config: p2p.Config{
 			PrivateKey:  key,
-			DiscoveryV5: true,
 			NoDiscovery: true,
 		},
 	}
 	require.NoError(t, server.Start())
 	defer server.Stop()
-	require.NotNil(t, server.DiscV5)
+
+	discovery := NewDiscV5(key, server.ListenAddr, nil)
+	require.NoError(t, discovery.Start())
+	defer func() { _ = discovery.Stop() }()
```

Comment:
We can check this with `require.NoError()`. I don't see a reason to avoid doing that.